### PR TITLE
Feature 3DS2

### PR DESCRIPTION
**Pull request by maassenbas**
Feature 3DS2

### CODE REVIEW

File: cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/storedPaymentInstruments.isml [hunk 1]
```diff
@@ -3,6 +3,7 @@
     assets.addCss('/css/adyen-checkout-2.1.0.css');
```
Comment (Aleffio) [inline cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/storedPaymentInstruments.isml]: please update to 3.0.0

File: cartridges/int_adyen_overlay/cartridge/static/default/css/adyen-checkout-2.5.0.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.adyen-checkout__field{display:block;margin-bottom:16px}.adyen-checkout__field--error input{border-color:#d10244;color:#d10244}.adyen-checkout__field:last-child{margin-bottom:0}.adyen-checkout__helper-text,.adyen-checkout__label__text{color:#00112c;display:block;font-size:.81em;font-weight:400;line-height:13px;padding-bottom:8px}.adyen-checkout__helper-text{color:#687282}.adyen-checkout__label__text{transition:color .2s ease-out}.adyen-checkout__label--focused .adyen-checkout__label__text{color:#06f}.adyen-checkout__error-text,.adyen-checkout__label__text--error{align-items:center;color:#d10244;display:flex;font-size:.75em;font-weight:400;margin-top:4px}
```
Comment (Aleffio) [inline cartridges/int_adyen_overlay/cartridge/static/default/css/adyen-checkout-2.5.0.css]: do we need this 2.5.0 file?

File: cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds [hunk 3]
```diff
@@ -446,6 +476,46 @@ var __AdyenHelper: Object = {
...
+            jsonObject["browserInfo"] = browserInfo;
+            
+            jsonObject["browserInfo"].screenWidth = 1024;
+	        jsonObject["browserInfo"].screenHeight = 500;
+	        jsonObject["browserInfo"].colorDepth = 24;
+	        jsonObject["browserInfo"].userAgent = "Chrome";
+	        jsonObject["browserInfo"].timeZoneOffset = 0;
+	        jsonObject["browserInfo"].language = "nl-NL";
+	        jsonObject["browserInfo"].javaEnabled = true;
+	        jsonObject["browserInfo"].acceptHeader = "text/html";
```
Comment (Aleffio) [inline cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds]: please check browserinfo

File: cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js [hunk 4]
```diff
@@ -1,7 +1,12 @@
+var threeDS2utils = require('./threeds2-js-utils.js');
+
 const configuration = {
     locale: $('#currentLocale').val(), // Defaults to en_US
     originKey: originKey,
-    loadingContext: loadingContext
+    environment: loadingContext,
```
Comment (Aleffio) [inline cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js]: should be 'test' or 'live', according to 3.0.0 documentation, even if it works apparently with the url as well

File: cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js [hunk 5]
```diff
@@ -421,11 +436,12 @@ $('button[value="add-new-payment"]').on('click', function (e) {
...
 function setPaymentData() {
-    $('#adyenEncryptedCardNumber').val(card.paymentData.encryptedCardNumber);
-    $('#adyenEncryptedExpiryMonth').val(card.paymentData.encryptedExpiryMonth);
-    $('#adyenEncryptedExpiryYear').val(card.paymentData.encryptedExpiryYear);
-    $('#adyenEncryptedSecurityCode').val(card.paymentData.encryptedSecurityCode);
-    $('#cardOwner').val(card.paymentData.holderName);
+    $('#adyenEncryptedCardNumber').val(card.state.data.encryptedCardNumber);
+    $('#adyenEncryptedExpiryMonth').val(card.state.data.encryptedExpiryMonth);
+    $('#adyenEncryptedExpiryYear').val(card.state.data.encryptedExpiryYear);
+    $('#adyenEncryptedSecurityCode').val(card.state.data.encryptedSecurityCode);
```
Comment (Aleffio) [inline cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js]: should these values be "card.state.data.paymentMethod.encryptedSecurityCode" like here? https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/168/files#diff-81c46487cee32a4e6ebac5e05d313f83R88

File: cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js (hunk 4 above)
Comment (maassenbas) [reply cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js]: Good one, let me change that

File: cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds (hunk 3 above)
Comment (maassenbas) [reply cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds]: Yes, I will delete static values.

**Pull request merged**